### PR TITLE
US19209 - refactor cleanup

### PR DESCRIPTION
**Pull request by janssejl**

### CODE REVIEW

File: src/components/personalization/crds-recommended-content/crds-recommended-content.tsx [hunk 1]
```diff
@@ -40,8 +40,8 @@ export class CrdsRecommendedContent {
   }
 
   private getUser() {
-    if (!isAuthenticated()) return null;
-    return CrdsApolloService.apolloClient.query({ query: GET_USER }).then(success => {
+    if (!AuthService.isAuthenticated) return null;
+    return ApolloService.client.query({ query: GET_USER }).then(success => {
```
Comment (j-brammer) [inline src/components/personalization/crds-recommended-content/crds-recommended-content.tsx]: .then on new line

File: src/components/personalization/crds-recommended-content/crds-recommended-content.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ export class CrdsRecommendedContent {
   }
 
   private getLifeStages() {
-    return CrdsApolloService.apolloClient.query({ query: GET_LIFESTAGES }).then(success => {
+    return ApolloService.client.query({ query: GET_LIFESTAGES }).then(success => {
```
Comment (j-brammer) [inline src/components/personalization/crds-recommended-content/crds-recommended-content.tsx]: .then on new line

File: src/components/personalization/crds-tithe-challenge/crds-tithe-challenge.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ import { isAuthenticated } from '../../../global/authInit';
 })
 export class CrdsTitheChallenge {
   private analytics = window['analytics'];
-  private contentBlockHandler: ContentBlockHandler;
+  private contentBlockHandler: ContentBlockService;
```
Comment (j-brammer) [inline src/components/personalization/crds-tithe-challenge/crds-tithe-challenge.tsx]: handler => service

File: src/components/personalization/crds-tithe-challenge/crds-tithe-challenge.tsx [hunk 4]
```diff
@@ -35,13 +35,13 @@ export class CrdsTitheChallenge {
   }
 
   public async componentWillLoad() {
-    await CrdsApolloService.subscribeToApolloClient();
-    this.contentBlockHandler = new ContentBlockHandler(CrdsApolloService.apolloClient, 'tithe challenge');
+    await ApolloService.getClient();
+    this.contentBlockHandler = new ContentBlockService(ApolloService.client, 'tithe challenge');
```
Comment (j-brammer) [inline src/components/personalization/crds-tithe-challenge/crds-tithe-challenge.tsx]: handler => service

File: src/components/personalization/crds-tithe-challenge/crds-tithe-challenge.tsx [hunk 5]
```diff
@@ -51,15 +51,15 @@ export class CrdsTitheChallenge {
   }
 
   public getUser() {
-    return CrdsApolloService.apolloClient.query({ query: GET_USER_GROUPS }).then(response => {
+    return ApolloService.client.query({ query: GET_USER_GROUPS }).then(response => {
```
Comment (j-brammer) [inline src/components/personalization/crds-tithe-challenge/crds-tithe-challenge.tsx]: .then new line

File: src/components/personalization/crds-tithe-challenge/crds-tithe-challenge.tsx [hunk 6]
```diff
@@ -82,7 +82,7 @@ export class CrdsTitheChallenge {
   }
 
   private getFeelingResponses() {
-    return CrdsApolloService.apolloClient.query({ query: GET_FEELING_RESPONSES }).then(response => {
+    return ApolloService.client.query({ query: GET_FEELING_RESPONSES }).then(response => {
```
Comment (j-brammer) [inline src/components/personalization/crds-tithe-challenge/crds-tithe-challenge.tsx]: .then new line

File: src/components/sites/crds-site-happenings/site-happenings.tsx [hunk 7]
```diff
@@ -18,7 +18,7 @@ export class SiteHappenings {
   private sites: Site[] = [];
   private happenings: CrdsHappening[] = [];
   private user: CrdsUser = null;
-  private contentBlockHandler: ContentBlockHandler;
+  private contentBlockHandler: ContentBlockService;
```
Comment (j-brammer) [inline src/components/sites/crds-site-happenings/site-happenings.tsx]: handler => service

File: src/components/sites/crds-site-happenings/site-happenings.tsx [hunk 8]
```diff
@@ -39,8 +39,8 @@ export class SiteHappenings {
   }
 
   public async componentWillLoad() {
-    await CrdsApolloService.subscribeToApolloClient();
-    this.contentBlockHandler = new ContentBlockHandler(CrdsApolloService.apolloClient, 'site happenings');
+    await ApolloService.getClient();
+    this.contentBlockHandler = new ContentBlockService(ApolloService.client, 'site happenings');
```
Comment (j-brammer) [inline src/components/sites/crds-site-happenings/site-happenings.tsx]: handler -> service

File: src/components/sites/crds-site-happenings/tests/crds-site-happenings-graphQL.spec.tsx [hunk 9]
```diff
@@ -17,10 +17,10 @@ describe('<crds-site-happenings> GraphQL I/O', () => {
...
     const authToken = await getSessionID(user_with_site.email, user_with_site.password);
     window['apolloClient'] = new ReplaySubject();
-    authInit(authToken);
-    await CrdsApolloService.subscribeToApolloClient();
-    this.happenings.CrdsApolloService = CrdsApolloService; 
-    this.happenings.contentBlockHandler = new ContentBlockHandler(null, null);
+    AuthService.init(authToken);
+    await ApolloService.getClient();
+    this.happenings.ApolloService = ApolloService; 
+    this.happenings.contentBlockHandler = new ContentBlockService(null, null);
```
Comment (j-brammer) [inline src/components/sites/crds-site-happenings/tests/crds-site-happenings-graphQL.spec.tsx]: handler => service

File: src/components/sites/crds-site-happenings/tests/crds-site-happenings-render.spec.tsx [hunk 10]
```diff
@@ -1,13 +1,13 @@
...
+import { ContentBlockService } from '../../../../shared/services/content-blocks/contentBlock.service';
 import { ReplaySubject } from 'rxjs';
+import { AuthService } from '../../../../shared/services/auth.service';
+import { ApolloService } from '../../../../shared/services/apollo.service';
 
 describe('<crds-site-happenings> Render', () => {
   beforeEach(() => {
     this.happenings = new SiteHappenings();
-    this.happenings.contentBlockHandler = new ContentBlockHandler(null, null);
+    this.happenings.contentBlockHandler = new ContentBlockService(null, null);
```
Comment (j-brammer) [inline src/components/sites/crds-site-happenings/tests/crds-site-happenings-render.spec.tsx]: handler => service

File: src/components/sites/crds-site-select/crds-site-select.tsx [hunk 11]
```diff
@@ -2,18 +2,18 @@ import { Component, Prop, State, Element, h, EventEmitter, Event, Listen } from
...
+import { ApolloService } from '../../../shared/services/apollo.service';
 
 @Component({
   tag: 'crds-site-select',
   styleUrl: 'crds-site-select.scss',
   shadow: true
 })
 export class CrdsSiteSelect {
-  private contentBlockHandler: ContentBlockHandler;
+  private contentBlockHandler: ContentBlockService;
```
Comment (j-brammer) [inline src/components/sites/crds-site-select/crds-site-select.tsx]: handler -> service

File: src/components/sites/crds-site-select/crds-site-select.tsx [hunk 12]
```diff
@@ -31,16 +31,16 @@ export class CrdsSiteSelect {
...
   }
 
   public async componentWillLoad() {
     this.initToastr();
-    await CrdsApolloService.subscribeToApolloClient();
+    await ApolloService.getClient();
     this.cookieSiteId = Number(Utils.getCookie('nearestSiteId'));
-    this.contentBlockHandler = new ContentBlockHandler(CrdsApolloService.apolloClient, 'my site');
-    return Promise.all([isAuthenticated() ? this.getUserSite() : null, this.contentBlockHandler.getCopy()]);
+    this.contentBlockHandler = new ContentBlockService(ApolloService.client, 'my site');
```
Comment (j-brammer) [inline src/components/sites/crds-site-select/crds-site-select.tsx]: handler -> service

File: src/global/app.ts [hunk 13]
```diff
@@ -1,5 +1,8 @@
-import { ReplaySubject } from 'rxjs';
-import { authInit } from './authInit';
+import { AuthService } from '../shared/services/auth.service';
+import { ApolloService } from '../shared/services/apollo.service';
 
-window['apolloClient'] = new ReplaySubject();
-authInit();
+
+AuthService.init();
+AuthService.watch().subscribe(token => {
```
Comment (j-brammer) [inline src/global/app.ts]: subscribe new line

File: src/components/connect/groups/crds-group-renew-button/crds-group-renew-button.tsx [hunk 14]
```diff
@@ -86,7 +86,7 @@ export class CrdsGroupRenew {
   }
 
   public render() {
-    if (!isAuthenticated() && !this.userLedGroups.find(group => Number(group.id) == this.groupId) || !this.groupId) return '';
+    if (!AuthService.isAuthenticated && !this.userLedGroups.find(group => Number(group.id) == this.groupId) || !this.groupId) return '';
```
[the remark this answers is not in the document]
Comment (janssejl) [reply src/components/connect/groups/crds-group-renew-button/crds-group-renew-button.tsx]: not related to this refactor.

File: src/global/app.ts (hunk 13 above)
Comment (janssejl) [reply src/global/app.ts]: why?

Comment (j-brammer) [reply src/global/app.ts]: it shouldn't. nothing to see here.

**Pull request merged**